### PR TITLE
Updated Jinjiang's personal information

### PR DESCRIPTION
**Pull request by Jinjiang**
Updated some information about myself:
- Not work for Alibaba / maintain Weex any more, for a while actually.
- Moved to Singapore from China.
- Added some ecosystem projects I personally created.
Thanks.

### CODE REVIEW

File: src/v2/guide/team.md [hunk 1]
```diff
@@ -353,21 +353,16 @@ order: 803
...
-      },
       reposOfficial: [
-        'cn.vuejs.org'
+        'cn.vuejs.org', 'vue-docs-zh-cn'
       ],
       reposPersonal: [
-        'apache/incubator-weex'
-      ]
+        'vue-a11y-utils', 'vue-mark-display', 'mark2slides', 'vue-keyboard-over'
+      ],
```
Comment (phanan) [inline src/v2/guide/team.md:365]: I think this will break in IE11.

**Comment by phanan** [comment]
Would you update docs-next with the new info as well?

**Comment by Jinjiang** [comment]
Sure. Will do the same thing :-)

**Comment by phanan** [comment]
👍

**Pull request merged**